Parser/asdl_c.py

```diff
@@ -628,6 +628,9 @@ class PyTypesVisitor(PickleVisitor):
 
     def visitModule(self, mod):
         self.emit("""
+_Py_IDENTIFIER(_fields);
+_Py_IDENTIFIER(_attributes);
+_Py_IDENTIFIER(index);
```

Comment:
If you look out to other usages for methods, you can see that we use `astmodulestate_global` to keep that references. For consistency with other parts of program, IMHO you should add these to there. (Don't repeat things like `_fields` that already present in state)

Reply:
Thanks, I removed them.